src/components/ObjectInfo.js

```diff
@@ -0,0 +1,35 @@
+function ObjectInfo({arrangement, onItemPress, side, style}) {
+
+    function onPress({idbtn, gender:gender }) {
+        onItemPress({side, id:idbtn, gender:gender});
+    }
+
+    return (
+        <li className={'UserInfo'+side} style={style}>
+            <div className="Flex-container">
+                <div className="Flex-info-container">
+                    <div className="UserFName">
```

Comment:
<img width="895" alt="image" src="https://user-images.githubusercontent.com/41253957/211663371-982e1892-ea86-4f60-9dfd-2e3cedafb08d.png">

https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

```suggestion
                    <div className="user-f-name">
```